lib/syncthing/syncthing.go

```diff
@@ -89,25 +88,29 @@ func New(cfg config.Wrapper, ll *db.Lowlevel, evLogger events.Logger, cert tls.C
 		stop:     make(chan struct{}),
 		stopped:  make(chan struct{}),
 	}
+	close(a.stopped) // Hasn't been started, so shouldn't block on Wait.
+	return a
 }
 
-// Run does the same as start, but then does not return until the app stops. It
+// Run does the same as Start, but then does not return until the app stops. It
 // is equivalent to calling Start and then Wait.
+// Either Run or Start may be called once only.
 func (a *App) Run() ExitStatus {
```

Comment:
Run() is unused, apart from a test that just checks that it does roughly what it says on the tin. I think we could do without it?